pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.black]
+line-length = 100
+include = '\.pyi?$'
+exclude = '''
+/(
+	\.git
+| \.hg
+| \.mypy_cache
+| \.tox
+| \.venv
+| _build
+| buck-out
+| build   
+)/ 
+'''
```

Comment:
Is it working even not having the reference of `pyproject.toml` in the `yaml`? Honest question.

Reply:
yes, pyproject.toml is used by black so when pre-commit check for black it will try to get config in folder and get pyproject.toml, same as flake8 config